specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2023-08-02-preview/CommonDefinitions.json

```diff
@@ -0,0 +1,72 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-08-02-preview",
+    "title": "Azure Spring Apps API Client"
```

Comment:
please update the title as the one in appplatform.json:https://github.com/Azure/azure-rest-api-specs/blob/main/specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2023-05-01-preview/appplatform.json#L5